src/patternfly/components/Divider/divider.scss

```diff
@@ -1,9 +1,23 @@
 .pf-c-divider {
   --pf-c-divider--Height: var(--pf-global--BorderWidth--sm);
   --pf-c-divider--BackgroundColor: var(--pf-global--BorderColor--100);
+  --pf-c-divider--m-vertical--Width: var(--pf-global--BorderWidth--md);
+  --pf-c-divider--m-vertical--MinHeight: var(--pf-global--spacer--md);
 
   height: var(--pf-c-divider--Height);
   background-color: var(--pf-c-divider--BackgroundColor);
   border: 0; // removes the default border styling on an hr
   flex-shrink: 0;
+
+  &.pf-m-vertical {
+    align-self: center;
+    width: var(--pf-c-divider--m-vertical--Width);
+    height: 100%;
+    min-height: var(--pf-c-divider--m-vertical--MinHeight);
```

Comment:
The CSS for this looks solid, but I'm not sure we want to impose a `min-height` on the divider as we don't know the context in which it will be used and it's something a user would need to override.

Reply:
I get that, but I believe that not including a `min-height` will lead to user confusion as they may think it's broken or doesn't work at all because they won't see anything unless it's put into the right context. This way, it may look incorrect, but at least it will be visible. And if inserted into an element with the needed height (but not an implicit layout height), it will still consume 100% height. It think it's a sound approach.

Reply:
> I believe that not including a min-height will lead to user confusion as developers may think it's broken or doesn't work at all because they won't see anything unless it's put into the right context. 

That's where the documentation should come in. I don't think this would actually solve a use case for the divider, it would only serve as a hint that they need to manage the height in some regard, but it comes with the cost of potentially imposing an unwanted height. If the documentation is clear, I don't think this is necessary.